pages/koneet-kiertoon.tsx

```diff
@@ -0,0 +1,411 @@
+import Head from 'next/head'
+import styled from 'styled-components'
+import { Content } from '../components/Content/Content'
+import { H1 } from '../components/Title/Title'
+import { Footer } from '../components/Footer/Footer'
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
+import { Gallery } from '../components/Gallery/Gallery'
+import { InfoBox } from '../components/InfoBox/InfoBox'
+
+const Section = styled.div`
+
+  margin-top: 3rem;
+  margin-bottom: 1rem;
+
+  h2 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1.8rem;
+  }
+
+  h3 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1.rem;
+  }
+
+  h4 {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins';
+    font-weight: 600;
+    font-size: 1rem;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+
+  svg {
+    vertical-align: middle;
+    margin-top: -3px;
+    opacity: 0.7;
+    margin-right: 0.3rem;
+  }
+`
+const Device = styled.div`
+
+  display: flex;
+  margin-top: 1rem;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: rgba(108, 108, 108, 0.09);
+  padding: 1rem;
+  padding: 1.5rem;
+  border-radius: 0.5rem;
+  box-sizing: border-box;
+
+  p, h3 {
+    width: 100%;
+  }
+
+  div {
+    width: 50%;
+    margin-top: 1rem;
+    display: flex;
+    justify-content: center;
+    align-items: stretch;
+    flex-wrap: wrap;
+  }
+
+  div > ul {
+    width: 100%;
+    margin-top: -1rem;
+  }
+
+  div > ul > li {
+    height: 70%;
+  }
+`;
+
+const DeviceAmount = styled.div`
+  text-transform: uppercase;
+  color: rgba(255,255,255,0.6);
+  width: auto !important;
+  text-align: left;
+  margin: 0 0 1rem 0 !important;
```

Comment:
Could you just use `margin-bottom: 1rem !important;`?

Reply:
Fixed in [e4125c5](https://github.com/Testausserveri/testausserveri.fi/pull/99/commits/e4125c56cf8cab9d9ec0c55e15c9f1b8876d5c51)